flink-connector/src/main/java/org/apache/flink/connector/delta/sink/DeltaSink.java

```diff
@@ -78,20 +79,31 @@
  * from FileSink.
  */
 public class DeltaSink<IN>
-        implements Sink<IN, DeltaCommittable, DeltaWriterBucketState, DeltaGlobalCommittable> {
+    implements Sink<IN, DeltaCommittable, DeltaWriterBucketState, DeltaGlobalCommittable> {
 
     private final DeltaSinkBuilder<IN> sinkBuilder;
 
     DeltaSink(DeltaSinkBuilder<IN> sinkBuilder) {
         this.sinkBuilder = checkNotNull(sinkBuilder);
     }
 
+    /**
+     * This method creates the {@link SinkWriter} instance that will be responsible for passing
+     * incoming stream events to the correct bucket writer and then flushed to the underlying files.
+     * <p>
```

Comment:
nit: no need for this `<p>` here, unless in future PRs there will be more text below.